examples/prospector.rb

```diff
@@ -1,8 +1,9 @@
 require 'clearbit'
-require 'pp'
 
-people = Clearbit::Prospector.search(domain: 'clearbit.com')
+response = Clearbit::Prospector.search(domain: 'clearbit.com')
```

Comment:
should we showcase here that we can pass a `page:` param in, even if its just `1`?